### PR TITLE
Create cmd_data_plots.py

### PR DESCRIPTION
**Pull request by MOchiara**

### CODE REVIEW

**Comment by MOchiara** [comment]
@callumrollo

File: cmd_data_plots.py [hunk 1]
```diff
@@ -0,0 +1,172 @@
+def basic_load(path):
```
Comment (callumrollo) [inline cmd_data_plots.py:18]: should there be some imports up here?

File: cmd_data_plots.py [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+    bla = np.diff(pd.to_datetime(a.DATE_TIME))
+    minutes = (bla / np.timedelta64(1000000000, 'ns')) / 60
+    a.loc[:, 0] = pd.to_datetime(a.DATE_TIME)
+    return a, minutes
+
+
+def make_all_plots(path_to_cmdlog):
+    active_m1 = load_cmd(path_to_cmdlog)
+    drift_dist, drift_y, drif_x, speed, u_vel, v_vel, theta, time, tot_time = measure_drift(active_m1)
+    print('Drift computed. Starting with dst')
```
Comment (callumrollo) [inline cmd_data_plots.py]: this will be automated, so use logging rather than print statements

**Review by callumrollo — CHANGES_REQUESTED** [review]

**Comment by MOchiara** [comment]
@callumrollo

**Review by callumrollo — APPROVED** [review]

**Pull request merged**